doc/sphinx/practical-tools/coqide.rst

```diff
@@ -209,10 +209,13 @@ The last section is for miscellaneous boolean settings, such as the
 “contextual menu on goals” feature presented in the section
 :ref:`Try tactics automatically <try-tactics-automatically>`.
 
-Notice that these settings are saved in the file `.coqiderc` of your
-home directory.
+Notice that these settings are saved in the file ``coqiderc`` in the
+``coq`` subdirectory of the user configuration directory which
+is the value of ``$XDG_CONFIG_HOME`` if this environment variable is
+set and which otherwise is ``$HOME/.config/``.
 
-A Gtk2 accelerator keymap is saved under the name `.coqide.keys`. It
+A Gtk2 accelerator keymap is saved under the name ``coqide.keys`` in
```

Comment:
Is it still "gtk2"?

Reply:
No... 

Then, I just changed it to "GTK+ accelerator keymap".